apps/web/pages/v2/apps/index.tsx

```diff
@@ -0,0 +1,58 @@
+import { InferGetStaticPropsType, NextPageContext } from "next";
+
+import { getAppRegistry, getAppRegistryWithCredentials } from "@calcom/app-store/_appRegistry";
+import { getSession } from "@calcom/lib/auth";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import prisma from "@calcom/prisma";
+import type { AppCategories } from "@calcom/prisma/client";
+import AllApps from "@calcom/ui/v2/core/apps/AllApps";
+import AppStoreCategories from "@calcom/ui/v2/core/apps/Categories";
+import AppsLayout from "@calcom/ui/v2/core/layouts/AppsLayout";
+
+export default function Apps({ appStore, categories }: InferGetStaticPropsType<typeof getServerSideProps>) {
+  const { t } = useLocale();
+
+  return (
+    <AppsLayout isPublic heading={t("app_store")} subtitle={t("app_store_description")}>
+      {/*<AppStoreCategories categories={categories} />*/}
```

Comment:
Why is this commented out? Can you remove to add a comment why it's commented out?

Reply:
This will be coming in a follow up: #4126